examples/color/colormap_reference.py

```diff
@@ -0,0 +1,94 @@
+"""
+==================
+Colormap reference
+==================
+
+Reference for colormaps included with Matplotlib.
+
+This reference example shows all colormaps included with Matplotlib. Note that
+any colormap listed here can be reversed by appending "_r" (e.g., "pink_r").
+These colormaps are divided into the following categories:
+
+Sequential:
+    These colormaps are approximately monochromatic colormaps varying smoothly
+    between two color tones---usually from low saturation (e.g. white) to high
+    saturation (e.g. a bright blue). Sequential colormaps are ideal for
+    representing most scientific data since they show a clear progression from
+    low-to-high values.
+
+Diverging:
+    These colormaps have a median value (usually light in color) and vary
+    smoothly to two different color tones at high and low values. Diverging
+    colormaps are ideal when your data has a median value that is significant
+    (e.g.  0, such that positive and negative values are represented by
+    different colors of the colormap).
+
+Qualitative:
+    These colormaps vary rapidly in color. Qualitative colormaps are useful
+for
+    choosing a set of discrete colors. For example::
+
+        color_list = plt.cm.Set3(np.linspace(0, 1, 12))
+
+    gives a list of RGB colors that are good for plotting a series of lines on
+    a dark background.
+
```

Comment:
Is it worth adding a section (maybe in another PR?) that lists all colorblind-friendly colormaps?

Reply:
I'm all for that/think that's a great idea.

Reply:
I don't think this should be added to this example. This is a reference list. This should be (if it isn't already) in the colormap tutorial.

Reply:
Why shouldn't a reference list provide accessibility references?

Reply:
Can this be handled by including a reference here to the tutorial section?

Reply:
I think so, so long as there's an explicit mention of colorblind maps, something "See the tutorial for a more detailed explanation of colormaps, including information on colorblind friendly"

Reply:
I would just delete *all* the text (it's a reference!) and replace it by "for an in-depth overview of colomaps, including discussion about colorblind-friendliness, see (tutorial)".

Reply:
There is no information on colorblind maps in the tutorial, just outside references.

Feel free to merge or close if you feel this is not an improvement to the documentation.